scripts/openstack.py

```diff
@@ -296,6 +292,90 @@ def get_parser():
         ],
         conflict_handler='resolve',
         add_help=False,
+        epilog="""test repos:
+
+Test repository can be specified using --test-repo optional argument
+with value in the following formats:  NAME:URL, NAME!PRIORITY:URL
+or @FILENAME. See examples:
+
+1) Essential usage requires to provide repo name and url:
+
+    --test-repo foo:http://example.com/repo/foo
+
+2) Repo can be prioritized by adding a number after '!' symbol
+   in the name:
+
+    --test-repo bar!10:http://example.com/repo/bar
```

Comment:
Should this be: `--test-repo 'bar!10:http://example.com/repo/bar'` (with single quotes because without them the shell attempts to evaluate the `!`, which fails and returns an error) ?

Reply:
you're right

Reply:
done